Uml.Robotics.Ros.MessageBase/TypeRegistryBase.cs

```diff
@@ -7,6 +7,9 @@
 using System.Runtime.Loader;
 using System.Text;
 
+#if NETCORE
+using System.Runtime.Loader;
```

Comment:
Can this block be removed?  The `using System.Runtime.Loader;` line is already unconditionally in line 7.

Reply:
sorry, it's a typo, I forgot to remove line 7